MeshKernelNET/src/MeshKernelNETCore/Api/MakeGridParameters.cs

```diff
@@ -7,31 +7,33 @@ namespace MeshKernelNETCore.Api
     [ProtoContract(AsReferenceDefault = true)]
     public class MakeGridParameters : INotifyPropertyChanged
     {
-        private double yGridBlockSize;
-        private double xGridBlockSize;
-        private double originZCoordinate;
-        private double originYCoordinate;
-        private double originXCoordinate;
-        private double gridBlockSize;
-        private double gridAngle;
         private int numberOfRows;
         private int numberOfColumns;
+        private double gridAngle;
+        private double originXCoordinate;
+        private double originYCoordinate;
+        private double xGridBlockSize;
+        private double yGridBlockSize;
+        private double upperRightXCoordinnate;
+        private double upperRightYCoordinnate;
+
+
         private GridTypeOptions gridType;
 
         public static MakeGridParameters CreateDefault()
         {
             return new MakeGridParameters
             {
                 GridType = GridTypeOptions.Square,
-                NumberOfColumns = 3,
-                NumberOfRows = 3,
+                NumberOfColumns = 0,
+                NumberOfRows = 0,
                 GridAngle = 0.0,
-                GridBlockSize = 0.0,
                 OriginXCoordinate = 0.0,
                 OriginYCoordinate = 0.0,
-                OriginZCoordinate = 0.0,
-                XGridBlockSize = 10.0,
-                YGridBlockSize = 10.0
+                XGridBlockSize = 0.0,
+                YGridBlockSize = 0.0,
+                UpperRightXCoordinate = 0.0,
+                UpperRightYCoordinate = 0.0
```

Comment:
Buy the same argument given for having non-zero number of rows and columns and grid spacing, it also makes sense to have upper right grid pint coordinates that are different than the origin, otherwise the mesh boils down to a point. Any suggestions for  meaningful values? Seems like anything goes here.

Reply:
Seems like leaving the coords at 0, an exception will be thrown. No further action needed.